### PR TITLE
Add GLU activation function

### PR DESCRIPTION
**Pull request by davidmascharka**
Some craziness in the test because of the conditions this thing requires :grimacing: 

Last one! :partying_face: :tada:

### CODE REVIEW

File: src/mygrad/nnet/activations/glu.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+from numpy import ndarray
+
+from mygrad import multiply, Tensor
+from .sigmoid import sigmoid
+
+
+def glu(x, dim=-1, constant=False):
```
Comment (rsokl) [inline src/mygrad/nnet/activations/glu.py:7]: I think this should be `axis` to stick with the parlance of NumPy

Comment (davidmascharka) [reply src/mygrad/nnet/activations/glu.py]: :flushed: why did I do that? :facepalm: updated

**Comment by davidmascharka** [comment]
@rsokl I didn't see a changelog - are we tracking changes? Was planning to bump to 1.6 and highlight that all these activations have been added

**Comment by rsokl** [comment]
@davidmascharka I am planning to generate the `.rst` docs for these, then I'll bump to 1.6

**Pull request merged**